### PR TITLE
Plural of parenthesis is parentheses

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
see #2779, which this PR is a follow-up to

### CODE REVIEW

**Comment by IsaacWeiss** [comment]
Maybe while at it also change the UI-visible strings?

**Comment by Jojo-Schmitz** [comment]
I thought I did.  Checking again...

**Comment by Jojo-Schmitz** [comment]
OK, tell me which I missed, I don't see any

**Comment by IsaacWeiss** [comment]
I was thinking the tooltip in the Noteheads palette, and the Inspector toggle for accidentals.

**Comment by Jojo-Schmitz** [comment]
The tooltips are taken care of, The popup in inspector for Bracket type is singular, None/Bracket/Parenthesis

**Comment by IsaacWeiss** [comment]
Aha, I missed 48f2f089!

**Comment by IsaacWeiss** [comment]
Though I'd urge plural for the Inspector, too...

**Comment by Jojo-Schmitz** [comment]
`Bracket type` looks right to me and matches the use of singular in the pull down menu. `Bracket**s** type` and chaning the pull down ones looks wrong to me.

File: mscore/shortcut.cpp [hunk 1]
```diff
@@ -3246,11 +3246,11 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
-         "add-parenthesis",
```
Comment (lasconic) [inline mscore/shortcut.cpp]: This would break existing preferences, so it can't go to 2.1

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: Hmm, I didn't think of this to possibly apply to 2.1. Should I?

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: Hmm, I didn't think of this to possibly apply to 2.1. Should I?

I think the differentiaton between brackets [] and parentheses {} is new in master

Comment (lasconic) [inline mscore/shortcut.cpp]: No. All fine.

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: What puzzlerd me though is that there has some compat code been added to read114() and read206(), as if it had been possible to use `[]` in 1.x and 2.x, see e.g. https://github.com/musescore/MuseScore/commit/48f2f0897f535085559c12aa2ceec1df1d634555#diff-9c93d8fdd487e712679e845aba7e30b6R245

**Pull request merged**